client-v2/src/shared/actions/Cards.ts

```diff
@@ -198,8 +208,24 @@ const getArticleEntitiesFromDrop = async (
   const isPlainUrl = isURL && !id && !guMeta;
   const isCAPIUrl = isCapiUrl(resourceIdOrUrl);
   if (isCAPIUrl) {
-    const card = await createAtomSnap(resourceIdOrUrl);
-    return [card];
+    try {
+      const atom = await getAtomFromCapi(
+        getAbsolutePath(resourceIdOrUrl, false)
+      );
+      const card = await createAtomSnap(resourceIdOrUrl, atom);
+      return [card];
+    } catch (e) {
+      dispatch(
+        startOptionsModal(
+          'Invalid atom link',
+          "It looks like you've tried to add an interactive atom that doesn't exist. Check the link and try again.",
```

Comment:
One thought on the message - all content-api urls that get pasted in here are funnelled into this modal. Perhaps we should clarify a bit more. 
```
"It looks like you've tried to add something from our content-api that we don't accept. Only interactive atoms can be added as cards. Check the link is for an interactive atom and that the link is valid and try again.
```

Reply:
Good point, I hadn't thought of that.